### PR TITLE
UG: fix conversion problems

### PR DESCRIPTION
**Pull request by jieffyy**
There's still many issues with the UG - seems messy.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -28,20 +27,9 @@ The target audience for Hall-y is the NUS Eusoff Hall administrative staff, who
...
 
-## 3 About This Document
-
-The following table explains the formatting used in this guide and its definition.
-
-Format | Definition, Examples
---------|------------------
-`code` | Represent a command or part of a command. <br> e.g. `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 br/A104 g/M m/A0199242X`
-UPPER_CASE | Represent a parameter to be supplied by the user. <br> e.g. `n/NAME` can be used as `n/John Doe`
-[item] | Represent an optional parameter to be supplied by the user. <br> e.g. `n/NAME [p/PHONE_NUMBER]` can be used as `n/John Doe p/91234567` or as `n/John Doe`
```
Comment (jieffyy) [inline docs/UserGuide.md:44]: This section is repeated.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -257,7 +203,6 @@ The steps for this command are as follows:
 
 Format: `export`
 
-
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the command format:**<br>
```
Comment (yanchenglee98) [inline docs/UserGuide.md]: This looks out of place

**Review by yanchenglee98 — COMMENTED** [review]
After export feature, there is a section that seems to be at the wrong place.

File: docs/UserGuide.md (hunk 2 above)
Comment (jieffyy) [reply docs/UserGuide.md]: Thanks, fixed.

**Review by yanchenglee98 — APPROVED** [review]
Will need to update in the future

**Pull request merged**